docs/source/commands/persisted-queries.mdx

```diff
@@ -53,4 +53,4 @@ rover persisted-queries publish \
 
 2. Updates any _other_ variants that the PQL is applied to so that routers associated with those variants can fetch their updated PQL.
 
-> As with [generating manifests](/graphos/operations/persisted-queries#31-generate-persisted-queries-manifests), it's best to execute this command in your CI/CD pipeline to publish new operations as part of your app release process.
\ No newline at end of file
+> As with [generating manifests](/graphos/operations/persisted-queries#31-generate-persisted-queries-manifests), it's best to execute this command in your CI/CD pipeline to publish new operations as part of your app release process. This command requires your API key to have the [**Graph Admin** or **Persisted Query Publisher** role](/graphos/org/members#graph-api-key-roles); the latter is a role specifically designed to work with this command without exposing unnecessary information about your graph to users of the API key.
```

Comment:
Same comment as on the graphOS docs: wouldn't the required API key roles be better placed closer to the beginning?

Reply:
Didn't know graph admin key as default so resolving this!